Index: my-application/.editorconfig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-application/.editorconfig b/my-application/.editorconfig
new file mode 100644
--- /dev/null	(date 1649357334587)
+++ b/my-application/.editorconfig	(date 1649357334587)
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+trim_trailing_whitespace = true
+insert_final_newline = false
+
+[*.md]
+trim_trailing_whitespace = false
\ No newline at end of file